Add Talos osctl command to plugins

particledecay opened the pull request:
Current build status: ![Build Status](https://travis-ci.org/particledecay/asdf-osctl.svg?branch=master)

Stratus3D commented:
Thanks for the PR!

Stratus3D commented:
One thing I noticed is that you are using `jq` to parse the available versions. The other plugins typicall use sed for parsing versions as sed is more widely available, and installed on all unix OSes. You might consider switching to sed to make it easier on your users.

particledecay commented:
@Stratus3D I just removed it. It wasn't really necessary for osctl, but I had already added it for etcdctl so I figured why not. Better to not include it if it's not needed though.
